superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeGallery.tsx

```diff
@@ -332,6 +333,38 @@ const thumbnailContainerCss = (theme: SupersetTheme) => css`
   }
 `;
 
+const BetaBadge = styled.div`
```

Comment:
I guess BetaBadge is OK, but we might want to have something more generic than Beta (which seems purposeful). Something like `HighlightLabel`?.

Reply:
Actually, I see you have `ThumbnailBadgeWrapper` later, so maybe `ThumbnailBadge` would be fitting :)

Reply:
Good idea. This component is used in two places, so `HighlightLabel` feels more generic, and the parent can use `ThumbnailLabelWrapper` and `TitleLabelWrapper`. wdyt?